city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+from datetime import datetime
+from difflib import SequenceMatcher
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa52Spider(CityScrapersSpider):
+    name = "chi_ssa_52"
+    agency = "Chicago Special Service Area #52 51st Street"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.51ststreetchicago.com/about.html"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items = response.css("div.paragraph")[3:4]
+        titles = items.css("strong::text").getall()
+        meetings = items.css("ul")
```

Comment:
We only want the first `ul` here since the second one isn't related to the Commission

Reply:
So i don't have to parse the `GENERAL BUSINESS MEETING DATES 2019` part?

Reply:
Yep, you can feel free to ignore it. These "Special Service Areas" are governmental bodies that are managed by private entities, and it looks like the "General Business Meeting" items are of the private entity

Reply:
Ok, thanks for the tip